### PR TITLE
RM-6664: Swallow postback requests triggered by double clicking a button or hyperlink

### PR DESCRIPTION
**Pull request by iam-mholle**
https://re-motion.atlassian.net/browse/RM-6664

Testable by hosting `Web.Test` -> `/MultiplePostbackCatching/UpdatePanelSutForm.aspx`.

### CODE REVIEW

**Review by Stefan-Ilic — APPROVED** [review]

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

File: Remotion/Web/Core/Res/HTML/SmartPage.js [hunk 1]
```diff
@@ -1164,6 +1165,11 @@ function SmartPage_Context(
       {
         isAutoPostback = true;
       }
+
+      if (tagName === 'a' || type === 'submit' || type === 'button')
```
[the remark this answers is not in the document]
Comment (iam-mholle) [reply Remotion/Web/Core/Res/HTML/SmartPage.js:1169]: Oh well, didn't know that - makes everything simpler tho, thanks for the explanation.

File: Remotion/Web/Core/Res/HTML/SmartPage.js [hunk 2]
```diff
@@ -1176,7 +1176,7 @@ function SmartPage_Context(
     };
     if (_submitState == null)
       _submitState = submitState;
-    else
+    else if (isAutoPostback || submitterElement.id !== _submitState.Submitter.id)
```
Comment (MichaelKetting) [inline Remotion/Web/Core/Res/HTML/SmartPage.js:1179]: submitterElement can be null, see line 1151

**Review by MichaelKetting — CHANGES_REQUESTED** [review]

**Review by MichaelKetting — APPROVED** [review]
I'm assuming this succeeded in a manual test. the code does look correct but I have learned to be careful in the web.

**Comment by iam-mholle** [comment]
Added to master with be835625ea6356be219932d1a0520a6e28fe8d7e.

**Pull request closed without merging**